contracts/modules/staking/StakingPool.sol

```diff
@@ -1298,24 +1316,29 @@ contract StakingPool is IStakingPool, ERC721 {
 
   function getPremium(
```

Comment:
we need to re-enable tests for the price sooner rather than later so we can refactor with confidence

it's fine to skip for now, i guess we'll fix it all together when tests are ported over

Reply:
Yes, we'll have to, but that's sort of out of scope right now because I didn't disable them in this PR. There's only one skipped test that's unrelated to the pricing itself.